helm_deploy/court-case-matcher/values.yml

```diff
@@ -0,0 +1,84 @@
+generic-service:
+  nameOverride: court-case-matcher
+
+  image:
+    repository: quay.io/hmpps/court-case-matcher
+    pullPolicy: Always
+    tag: app_version # override at deployment time
+    port: 8080
+
+  ingress:
+    enabled: true
+    modsecurity_enabled: false
+    hosts:
+      - app-hostname.local    # override per environment
+    path: /
+    annotations:
+      nginx.ingress.kubernetes.io/configuration-snippet: |
+        server_tokens off;
+        location /queue-admin/retry-all-dlqs {
+          deny all;
+          return 401;
+        }
+        location /replay404Hearings {
+          deny all;
+          return 401;
+        }
+
+  allowlist:
+    groups:
+      - internal
+
+  # Pre-existing kubernetes secrets to load as environment variables in the deployment.
+  # namespace_secrets:
+  #   [name of kubernetes secret]:
+  #     [name of environment variable as seen by app]: [key of kubernetes secret to load]
+
+  namespace_secrets:
+    crime-portal-gateway-s3-credentials:
+      CRIME_PORTAL_GATEWAY_S3_BUCKET: bucket_name
+    court-case-matcher-secrets:
+      OFFENDER_SEARCH_CLIENT_ID: nomis-oauth-client-id
+      PERSON_RECORD_SEARCH_CLIENT_ID: nomis-oauth-client-id
+      PERSON_RECORD_SEARCH_CLIENT_SECRET: nomis-oauth-client-secret
+      APPINSIGHTS_INSTRUMENTATIONKEY: APPINSIGHTS_INSTRUMENTATIONKEY
+      OFFENDER_SEARCH_CLIENT_SECRET: nomis-oauth-client-secret
+    court-case-matcher-queue-credentials:
+      AWS_SQS_COURT_CASE_MATCHER_QUEUE_NAME: sqs_name
+      AWS_SQS_COURT_CASE_MATCHER_ENDPOINT_URL: sqs_id
+    court-case-matcher-queue-dead-letter-queue-credentials:
+      AWS_SQS_COURT_CASE_MATCHER_DLQ_QUEUE_NAME: sqs_name
+      AWS_SQS_COURT_CASE_MATCHER_DLQ_ENDPOINT_URL: sqs_id
+
+  autoscaling:
+    enabled: true
+    targetCPUUtilizationPercentage: 100
+
+  env:
+    JAVA_OPTS: "-Xmx512m"
```

Comment:
It would be simpler to standardise this across all environments I think, no real reason not to go for the largest value